ui/src/routes/conversations/[id]/+page.svelte

```diff
@@ -150,6 +152,13 @@
     loadingMessagesOld = true;
     try {
       await messages.loadMessagesInPreviousBucketTargetCount();
+      await tick();
+      if (virtualList) {
```

Comment:
```suggestion
      if (!virtualList)
```
I prefer this pattern because it makes clear that we always expect virtuallist to be defined by the time this function is run.